mysql/mysqldump.go

```diff
@@ -238,13 +246,35 @@ func toSchemaKeys(columns []*ast.IndexPartSpecification) (keys []schema.Key) {
 	return keys
 }
 
+// toForeignKeys converts a string list of MySQL foreign keys to
```

Comment:
"converts a string list of MySQL foreign keys"
I'm not sure that's an accurate description -- fk isn't really a string list?
How about: "converts a MySQL ast foreign key constraint"

Reply:
updated comment